Altis_Life.Altis/stringtable.xml

```diff
@@ -1278,6 +1278,85 @@
             <Portuguese>Chamar SAMU</Portuguese>
             <Polish>MEDYK</Polish>
         </Key>
+	</Package>
```

Comment:
This line is indented using a tab. It should be indented via 4 spaces per tab as per [contributing](https://github.com/AsYetUntitled/Framework/blob/master/.github/CONTRIBUTING.md#help-the-travis-ci-build-failed). 

For an unknown reason [Travis CI did not catch this](https://travis-ci.org/AsYetUntitled/Framework/builds/189383459) as it passed. Ping, @BoGuu.